app/scripts/controllers/login.ctrl.js

```diff
@@ -15,17 +15,22 @@ angular.module('dashboard')
     var self = this;
     self.data = { selection : null, options : [] };
     self.meetings = [];
+    self.loadingUsers = true;
+    self.login = false;
 
     $http({
         method: 'GET',
         url: ENV.AhjoApi_UserLogin
     }).then(function successCallback(response) {
+        self.loadingUsers = false;
         self.data.options = angular.element(response.data).find('option');
     }, function errorCallback(error) {
         $log.error(error);
+        self.loadingUsers = false;
```

Comment:
This should use "finally" branch for common cleanups like "self.loadingUsers = false;"